docs/platforms/unreal/configuration/options.mdx

```diff
@@ -72,6 +84,12 @@ Grouping in Sentry is different for events with stack traces and without. As a r
 Takes a screenshot of the application when an error happens and includes it as an attachment.
 Learn more about enriching events with screenshots in our <PlatformLink to="/enriching-events/screenshots/">Screenshots documentation</PlatformLink>.
 
+<Alert level="info">
+
+The Unreal Engine SDK support for screenshot attachment on crash events is limited to Windows and Linux.
```

Comment:
Is it the Unreal Engine SDK as in, provided by Epic?
Or are we talking about the Sentry SDK for Unreal Engine? 
I think either way we could just say:

```suggestion
The support for screenshot attachment on crash events is limited to Windows and Linux.
```

We could add a link to:
* https://github.com/getsentry/sentry-unreal/issues/567

But tbqh screenshot on mobile is supported by the underlying SDK so should be a matter of passing the flag down:

https://github.com/getsentry/sentry-unreal/issues/567#issuecomment-2440085247

Reply:
Thanks, I'll revisit the screenshot attachments feature for Android/Apple. Last time I've checked things didn't seem to work out of the box for Unreal on mobile but maybe something has changed since then.